### PR TITLE
Fix: Fake --> Freak

### PR DESCRIPTION
**Pull request by 7ijme**
I had an issue where with the Bomb check, I couldn't tell the app "Freak no". It would interpret it as Fake, which didn't do anything. This fixes the issue.

### CODE REVIEW

**Review by GeorgeMC2610 — COMMENTED** [review]
For some reason, as of commit 8f2041c2a3b42575e2f116c2e889ee28ee0219dc, I reversed this. Please standby as I review this change.

File: Grammar/DefuseGrammar.cs [hunk 1]
```diff
@@ -216,4 +216,4 @@ private static Grammar _WhoIsOnFirstGrammar()
             return new Grammar(new GrammarBuilder(new Choices(File.ReadAllLines(@"WhoIsOnFirst.txt")))) {Name = "Who's On First Grammar"};
         }
     }
-}
```
Comment (GeorgeMC2610) [inline Grammar/DefuseGrammar.cs:219]: Any reason in particular for this whitespace? Or did you just forget it?

**Comment by GeorgeMC2610** [comment]
## Summary
Ok, so apparently, this change I made was kind of pointless, **because changing from `Freak` to `Fake` broke rest of the code,** and it didn't register the information about the FRK label. **I changed it and tried it myself,** so that you don't have to, and it is -in fact- a fix.

## History
I made this commit (8f2041c2a3b42575e2f116c2e889ee28ee0219dc) for #1. There was this person telling me that the computer wasn't willing to separate "Freak" and "Vowel" words. So I changed `Freak` to `Fake` as this was going to fix anything (spoiler alert: it didn't).

<br><br>
But anyway, thanks for your contribution! I will merge this as soon as possible.

File: Grammar/DefuseGrammar.cs (hunk 1 above)
Comment (7ijme) [reply Grammar/DefuseGrammar.cs:219]: No particular reason. My bad.

**Pull request merged**